DentOS_Framework/DentOsTestbedLib/src/dent_os_testbed/lib/os/linux/linux_sysctl_impl.py

```diff
@@ -0,0 +1,41 @@
+from dent_os_testbed.lib.os.linux.linux_sysctl import LinuxSysctl 
+class LinuxSysctlImpl(LinuxSysctl):
+    """
+        system kernel attributes manager
+    """
+    def format_get(self, command, *argv, **kwarg):
+        """
+        Get the attribute value
+        
+        """
+        params = kwarg["params"]
+        cmd = 'sysctl -n {} {}'.format(command, params["variable"])
```

Comment:
I think that using `-n` by default is wrong. What if I want to get a list of variables?:
```bash
$ sysctl net.ipv4
...
net.ipv4.conf.all.accept_local = 0
net.ipv4.conf.all.accept_redirects = 0
net.ipv4.conf.all.accept_source_route = 0
net.ipv4.conf.all.arp_accept = 0
net.ipv4.conf.all.arp_announce = 0
...
```